curriculum/challenges/english/02-javascript-algorithms-and-data-structures/functional-programming/understand-functional-programming-terminology.english.md

```diff
@@ -46,41 +46,31 @@ tests:
 <div id='js-seed'>
 
 ```js
-/**
- * A long process to prepare green tea.
- * @return {string} A cup of green tea.
- **/
+// Function that returns a string representing a cup of green tea
 const prepareGreenTea = () => 'greenTea';
 
-/**
- * A long process to prepare black tea.
- * @return {string} A cup of black tea.
- **/
+// Function that returns a string representing a cup of black tea
 const prepareBlackTea = () => 'blackTea';
 
-/**
- * Get given number of cups of tea.
- * @param {function():string} prepareTea The type of tea preparing function.
- * @param {number} numOfCups Number of required cups of tea.
- * @return {Array<string>} Given amount of tea cups.
- **/
+/*
+Given a function (representing the tea type) and number of cups needed, the
+following function returns an array of strings (each representing the a cup of
+a specific type of tea).
```

Comment:
```suggestion
Given a function, prepareTea (representing tea creation), and the number of cups 
needed, the following function returns an array of strings (each representing 
a cup of a specific type of tea).
```
Same as learn-about-functional-programming

Reply:
I will fix this in a separate commit.